backend/data/road_segmenting/calculate_distance.py

```diff
@@ -60,15 +60,15 @@ def calculate_road_length(gps_list, max_length_meter, haversine):
 
     length = 0
     index = 0
-    for gps_point in gps_list:
-        index += 1
-        if gps_list.index(gps_point) > 0:
-            prev = gps_list[gps_list.index(gps_point) - 1]
+    for i in range(len(gps_list)):
+        index = i
```

Comment:
`i` doesn't seem to be altered by anything but the for-loop, so why keep `index` here for the return when `i` has the same value?